modules/btcpayserver.nix

```diff
@@ -117,9 +117,9 @@ in {
       configFile = builtins.toFile "config" ''
         network=${config.services.bitcoind.network}
         btcrpcuser=${cfg.bitcoind.rpc.users.btcpayserver.name}
-        btcrpcurl=http://${config.services.bitcoind.rpcbind}:${toString cfg.bitcoind.rpc.port}
-        btcnodeendpoint=${config.services.bitcoind.bind}:8333
-        bind=${cfg.nbxplorer.bind}
+        btcrpcurl=http://${config.services.bitcoind.rpc.address}:${toString cfg.bitcoind.rpc.port}
+        btcnodeendpoint=${config.services.bitcoind.address}:8333
```

Comment:
Is the static port 8333 intentional (instead of `bitcoind.port`)?

Reply:
I had saved that for another PR (`Replace direct usage of 8332, 8333 with variables`), but you're right, this belongs here.(`fixup! use option value for btc port`)